asterisk/Dockerfile

```diff
@@ -187,27 +186,29 @@ RUN \
 
 ENV INSTALL_DIR="/opt/asterisk"
 
-RUN \
-    # 1.5 jobs per core works out okay \
-    JOBS=$(( $(nproc) + $(nproc) / 2 )); \
-    make -j ${JOBS} all; \
```

Comment:
Oh, ok got it. Restoring..

Reply:
What is a docker good practice here?

Reply:
I don't think there is any suggestion. And I'm okay, as long as it works.

I had removed because I wasn't sure what it was doing before. Now I do. Thanks.